python/lsst/pipe/tasks/makeWarp.py

```diff
@@ -573,12 +638,20 @@ def prepareCalibratedExposures(self, calExpList, backgroundList=None, skyCorrLis
         for index, (calexp, background, skyCorr) in enumerate(zip(calExpList,
                                                                   backgroundList,
                                                                   skyCorrList)):
```

Comment:
I think this should be zipping `wcsList` as well, instead of doing `wcsList[index]`, for consistency/clarity if nothing else (unless there's a reason I don't see for that).

Reply:
It's really only the one instance below in 
`if externalSkyWcsCatalog is None and wcsList[index] is None:`
that would get updated as I still need to use the `wcsList[index]` to assign the new external calibrations, but if you think that's still worth it, I'm happy to add it to the zip list!